pre-stop.d/10_remove_legacy_push_mobile.sh

```diff
@@ -2,23 +2,29 @@
 # Copyright 2019 The Wazo Authors  (see the AUTHORS file)
 # SPDX-License-Identifier: GPL-3.0-or-later
 
-sentinel="/var/lib/xivo-upgrade/10_remove_legacy_push_mobile"
+set -euo pipefail
+
+sentinel="/var/lib/xivo-upgrade/10_remove_legacy_push_mobile_v2"
 
 [ -e "$sentinel" ] && exit 0
 
 # Remove legacy configuration, everything is preconfigured
 # within each services
 rm -f /etc/wazo-calld/conf.d/push-mobile.yml \
       /etc/wazo-webhookd/conf.d/mobile.yml \
-      /etc/wazo-webhookd/conf.d/51-wazo-push-mobile.yml \
       /etc/wazo-auth/conf.d/mobile.yml
```

Comment:
the rm -f is now useless with purge

Reply:
Well I haven't found how the configuration become part of the debian package.
But ok, I tested it and purge removed them, so.

Reply:
Need a little bit of `poudre verte` and tadam :laughing: